ts/image-occlusion/MaskEditor.svelte

```diff
@@ -44,6 +48,12 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
             });
         }
     }
+
+    onMount(() => {
+        window.onresize = () => {
+            setCanvasZoomRatio(canvas, instance);
```

Comment:
A couple of tidy-ups please:

- addEventListener('resize', ...) would be better, as window.onresize could cause conflicts with other code down the line
- The listener should be removed when the component is unmounted, which you can do with onDestroy